templates/htan-synapse-s3-storage.yaml

```diff
@@ -0,0 +1,95 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: S3Objects
+Description: 'Synapse S3 Custom Storage (https://docs.synapse.org/articles/custom_storage_location.html)'
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: S3 Bucket Policy Configuration
+        Parameters:
+          - BucketName
+          - SynapseUserName
+          - S3UserARNs
+Parameters:
+  SynapseUserName:
```

Comment:
We're supporting userid/teamid and encourage migrating to it. Could we renam param and change description to Synapse userId or teamId. Technically we also support multiple but I think that would complicate more than it's worth.

Reply:
here's the issue @xschildw https://sagebionetworks.jira.com/browse/IT-831

would you like to work on it?